etl/steps/data/garden/health/2022-12-28/karlinsky.meta.yml

```diff
@@ -0,0 +1,50 @@
+dataset:
+  sources:
+  #   -
+  #     name: Maddison Project Database 2020 (Bolt and van Zanden, 2020)
+  #     published_by: Bolt, Jutta and Jan Luiten van Zanden (2020), “Maddison style estimates of the evolution of the world economy. A new 2020 update“.
+  #     publisher_source: |
+  #       The Maddison Project Database is based on the work of many researchers that have produced estimates of...
+
+tables:
+  deaths_karlinsky:
```

Comment:
Why not just calling the table `deaths`. It seems unnecessary to include the source in the table's name.

Reply:
Makes sense!